src/update_ram.c

```diff
@@ -42,20 +52,33 @@ void RAMFUNCTION wolfBoot_start(void)
 {
     int active, ret = 0;
     struct wolfBoot_image os_image;
-    uint32_t* load_address = (uint32_t*)WOLFBOOT_LOAD_ADDRESS;
+    uint32_t* load_address = (uint32_t*)kernel_addr;
```

Comment:
Please use `PLATFORM_X86_64_EFI` macro to keep existing behavior. This file is used by other platforms too.

Reply:
Fixed in 0e23b4adfe61fbac12841614728f09510905aaf1